test_apps/advection/README.md

```diff
@@ -0,0 +1,56 @@
+# VAPOR Particle Advection Benchmarking
+
+This directory has the benchmarking program for VAPOR's particle advection module
+
+## Build project
+
+The project has 1 external dependency apart from VAPOR
+-- The C++ Boost library [Program Options, File System]
+
+In order to find all the required VAPOR components the following variables must
+be provided to CMake
+
+```
+VAPOR_PREFIX : path to the VAPOR build/installation
+VAPOR_INCLUDE_DIR : path to the include directory in the VAPOR repository
+VAPORDEPS_PREFIX : path to where the VAPOR dependencies are located
+```
+A sample CMake command to build this project would look like
+
+```
+cmake -DVAPOR_PREFIX=/Users/abhishek/repositories/VAPOR/build \
+      -DVAPOR_INCLUDE_DIR=/Users/abhishek/repositories/VAPOR/include \
+      -DVAPORDEPS_PREFIX=/usr/local/VAPOR-Deps/2019-Aug \
+      -DCMAKE_BUILD_TYPE=Release \
+      .
+```
+
+## Running benchmarks
+
+The program requires an input config file for all the required parameters.
+A smaple file `demo10K.ini` is included in this repository.
+
+```
+datapath=/glade/u/home/abhisheky/NetCDF
```

Comment:
It looks like the data set you have in your home directory is one of our canonical test data sets from /glade/p/cisl/vast/vapor/data/Source/CM1/Orf/24Maycontrol 

If so, can you update the README to use the canonical version (it will have more permanency than your home directory).

Reply:
Ok, I'll update the path to the one already in the repository.